setup.cfg

```diff
@@ -16,18 +16,18 @@ classifiers =
 packages = find:
 python_requires = >=3.9
 install_requires =
-    pandas == 1.3.5
-    finta >= 1.3
-    pyyaml
-    tqdm
-    tzlocal >=3.0
-    tzdata
-    yfinance
-    SQLAlchemy
-    flask-login
-    flask-cors
-    flask
-    rich
+    pandas == 2.2.2
+    finta == 1.3
+    pyyaml == 5.3
+    tqdm == 4.66.4
+    tzlocal == 5.2
+    tzdata == 2024.1
```

Comment:
this will make it difficult for others to use this module. mb >= rather than == for the more general modules like pandas